### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by ydah**
This PR add some spelling corrections I detected below:
- https://github.com/rubocop/rubocop/pull/10937

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6748,6 +6752,7 @@ commndline->commandline
 commnds->commands
 commnent->comment
 commnents->comments
+commnet->comment
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:6755]: additional entries

```
commneted->commented
commneting->commenting
commnets->comments
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7271,6 +7276,7 @@ condfigures->configures
 condfiguring->configuring
 condict->conduct
 condicted->conducted
+condidate->candidate
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:7279]: `condidates->candidates`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -7826,6 +7832,7 @@ consitutute->constitute
 consitututed->constituted
 consitututes->constitutes
 consitututing->constituting
+consntant->constant
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:7835]: `consntants->constants`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -23072,6 +23087,7 @@ officealy->officially
 officialy->officially
 offloded->offloaded
 offred->offered
+offsense->offense
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:23090]: `offsenses->offenses`

File: codespell_lib/data/dictionary_en-GB_to_en-US.txt [hunk 5]
```diff
@@ -152,6 +152,8 @@ normalise->normalize
 normalised->normalized
 normalises->normalizes
 normalising->normalizing
+offence->offense
+offences->offenses
```
Comment (luzpaz) [inline codespell_lib/data/dictionary_en-GB_to_en-US.txt:156]: Please remove, `offence` is a valid GB spelling

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -23054,12 +23067,14 @@ offcers->officers
...
 offcially->officially
 offcials->officials
+offens->offense
 offerd->offered
 offereings->offerings
 offest->offset
 offests->offsets
 offet->offset, offer,
 offets->offsets, offers,
+offfense->offense
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:23077]: `offfenses->offenses`

**Review by luzpaz — CHANGES_REQUESTED** [review]

**Comment by ydah** [comment]
I updated this PR. Thank you so much.

**Comment by luzpaz** [comment]
@vikivivi care to weigh-in ?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:6755]:
```text
commnetaries->commentaries
commnetator->commentator
commnetators->commentators
```
* https://github.com/search?q=commnetaries&type=code  26
* https://github.com/search?q=commnetator&type=code  24
* https://github.com/search?q=commnetators&type=code  30

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:7839]:
```text
consntantly->constantly
```
https://github.com/search?q=consntantly&type=code  36

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -11254,6 +11268,7 @@ directort->directory
 directorty->directory
 directorys->directories
 directoty->directory
+directove->directive
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11271]:
```text
directoves->directives
```
https://github.com/search?q=directoves&type=code  595

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -19178,6 +19194,7 @@ interpolaiton->interpolation
 interpolar->interpolator
 interpolayed->interpolated
 interporated->interpolated, interpreted,
+interporation->interpolation
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:19197]:
```text
interporations->interpolations
```
https://github.com/search?q=interporations&type=code  32

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -20299,6 +20316,7 @@ locatins->locations
 loccked->locked
 locgical->logical
 lockingf->locking
+lodable->loadable
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:20319]:
```text
loded->loaded
loder->loader
loders->loaders
```
* https://github.com/search?q=loded&type=code  45512
* https://github.com/search?q=loder&type=code  107258
* https://github.com/search?q=loders&type=code  2916

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -23054,12 +23072,15 @@ offcers->officers
 offcial->official
 offcially->officially
 offcials->officials
+offens->offense
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:23075]: Please suggest `offend, offends, offers,` as well
```text
offens->offend, offense, offends, offers,
```
* https://github.com/search?q=offens&type=code  43919

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:23082]: Add `en-GB` as well
```text
offfence->offence
offfences->offences
```
* https://github.com/search?q=offfence&type=code  68
* https://github.com/search?q=offfences&type=code  33

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:23096]: Ignore 0 hit plural version `offsences->offences`
```text
offsence->offence
```
* https://github.com/search?q=offsence&type=code  16
* https://github.com/search?q=offsences&type=code  0

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -24727,6 +24751,7 @@ platflorm->platform
 platflorms->platforms
 platfoem->platform
 platfom->platform
+platfoms->platforms
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:24754]:
```text
platfomr->platform
platfomrs->platforms
```
* https://github.com/search?q=platfomr&type=code  9470
* https://github.com/search?q=platfomrs&type=code  760

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by ydah** [comment]
Thank you so much! I updated this PR.

**Review by vikivivi — APPROVED** [review]
LGTM

**Comment by luzpaz** [comment]
Thank you to all involved!

**Pull request merged**